lcc_lokavaluto_app_connection/models/account_invoice.py

```diff
@@ -0,0 +1,32 @@
+from odoo import models, fields, api
+
+
+class AccountInvoice(models.Model):
+    _inherit = "account.invoice"
+
+    has_numeric_lcc_products = fields.Boolean(
+        string="Has lcc numeric products",
+        compute="_compute_has_numeric_lcc_products",
+        store=True,
+    )
+
+    @api.one
+    @api.depends("state", "invoice_line_ids.product_id", "invoice_line_ids.price_total")
+    def _compute_has_numeric_lcc_products(self):
+        self.has_numeric_lcc_products = False
+
+        categ = self.env.ref(
+            "lcc_lokavaluto_app_connection.product_category_numeric_lcc"
+        )
+        lcc_numeric_products = self.invoice_line_ids.filtered(
+            lambda line: line.product_id.categ_id == categ
+            or line.product_id.categ_id.parent_id == categ
+        )
+        if lcc_numeric_products:
+            self.has_numeric_lcc_products = True
+
+    def _get_credit_requests(self, backend_keys):
```

Comment:
a small comment to help recall that these function are to be implemented by backends modules may be helpful.

Reply:
Ok done :)